draft-ietf-httpbis-digest-headers.md

```diff
@@ -469,7 +492,9 @@ The Digest header field provides a digest of the representation data.
 - or not at all contained in the message body.
 
 The resource is specified by the effective
-Request-URI and any cache-validator contained in the message.
+Request-URI,
+the HTTP semantics of the message
+and any cache-validator contained in the message.
```

Comment:
@LPardue I need some guidance here:

- iiuc cache validators are ETags & Last Modified [RFC2616 s. 13.3.3 ](https://tools.ietf.org/html/rfc2616#section-13.3.3)

it's not clear to me:

* how a cache validator specify a resource;
* how can we translate the sentence according to RFC723x

Reply:
sorry, not a caching expert so my opinion might not help much here.

Reply:
As @reschke had similar considerations in https://github.com/httpwg/http-extensions/pull/940#pullrequestreview-299297863

I asked him to help.

Reply:
I thought twice. Probably rfc 3230 refers to receiving a cached `Digest`.

`validators` do not identify the resource on the origin server 
 but it's cached representation. Let me know, R.